run_service.sh

```diff
@@ -581,7 +581,7 @@ directory="trader"
 service_repo=https://github.com/$org_name/$directory.git
 # This is a tested version that works well.
 # Feel free to replace this with a different version of the repo, but be careful as there might be breaking changes
-service_version="v0.12.5"
+service_version="v0.12.7"
```

Comment:
Is this correct, bumping from 0.12.5 to 0.12.7 straight?

Reply:
Yes, `v0.12.6` was just a bump.